kore/src/Kore/ASTVerifier/PatternVerifier.hs

```diff
@@ -138,7 +138,7 @@ lookupAlias symbolOrAlias = do
     aliasConstructor = symbolOrAliasConstructor symbolOrAlias
     aliasParams = symbolOrAliasParams symbolOrAlias
     leftDefinition def =
-        fmap getElementVariable
+        fmap (foldMapVariable id)
```

Comment:
Doesn't this forget which variables are element or set?

Reply:
Yes, it does, but we have this information when we verify (in `verifyApplyAlias`), we lookup through the `Context`s `IndexModule`, which has `SentenceAlias`es (whose `left` is `!(Application SymbolOrAlias (UnifiedVariable Variable))`).

I didn't think this information is required once the verification is done. If you think it is, I can easily change the `Internal.Alias` type's `left` member to be `[UnifiedVariable Variable]` instead of just `[Variable]`.